Mathlib/Algebra/Pointwise/Stabilizer.lean

```diff
@@ -242,8 +242,8 @@ variable {G : Type*} [CommGroup G] (s : Set G)
 @[to_additive (attr := simp)]
 lemma mul_stabilizer_self : s * stabilizer G s = s := by rw [mul_comm, stabilizer_mul_self]
 
-local notation " Q " => G ⧸ stabilizer G s
-local notation " q " => ((↑) : G → Q)
+local notation "Q" => G ⧸ stabilizer G s
```

Comment:
These changes align the pretty-printer with how these symbols are used in practice.